.github/workflows/linting-and-tests.yml

```diff
@@ -319,10 +319,9 @@ jobs:
     name: End to end tests
     uses: ./.github/workflows/e2e-tests.yml
     with:
-      # NOTE: don't run on > 10.0.3 until this issue is fixed. It appears that something was changed
-      # with the plugin-proxy that caused it to start throwing HTTP 502s from time to time
-      # https://raintank-corp.slack.com/archives/C01C4K8DETW/p1692279329797149
-      grafana-image-tag: 10.0.2
+      # TODO: fix issues with running e2e tests against Grafana v10.2.x and v10.3.x
+      grafana-image-tag: 10.1.7
```

Comment:
### TODO

Once merging update the "Require status checks to pass before merging" setting for our `dev` branch protection rules ([here](https://github.com/grafana/oncall/settings/branch_protection_rules/26218945)) to require `End to end tests / Grafana: 10.1.7` _not_ `End to end tests / Grafana: 10.0.2` (otherwise PR builds will hang waiting for the 10.0.2 job to run/pass)